### PR TITLE
Deleted travis files and updated database.yml

### PR DESCRIPTION
**Pull request by rizzkhan1**
### Context
Cleanup source code and fix inconsistencies with environment files

### Changes proposed in this pull request
Delete Travis build pipeline related files
Delete and rename environment settings files

### CODE REVIEW

File: config/settings/development.yml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 manage_backend:
```
Comment (dankmitchell) [inline config/settings/development.yml]: Revert the changes in this file

**Comment by dankmitchell** [comment]
Some environments to remove from https://github.com/DFE-Digital/manage-courses-support/blob/master/config/webpacker.yml

File: config/settings/development.yml (hunk 1 above)
Comment (rizzkhan1) [reply config/settings/development.yml]: done
